src/watcher.js

```diff
@@ -17,19 +18,21 @@ if (process.argv.length < 3) {
 
 const config = require(path.join('../config/', process.argv[2]))
 
+config.deployedBridgesRedisKey = process.env.DEPLOYED_BRIDGES_REDIS_KEY || 'deployed:bridges'
```

Comment:
I think we should include `DEPLOYED_BRIDGES_REDIS_KEY` and `MULTIPLE_BRIDGES_CONCURRENCY` on `.env.ERC_TO_ERC_MULTIPLE.example` and add a description on `README.md`